test/e2e/generators/capz.go

```diff
@@ -59,6 +59,5 @@ func expandCredVariables(stdout []byte, creds auth.Creds) []byte {
 	os.Setenv("AZURE_CLIENT_SECRET_B64", base64.StdEncoding.EncodeToString([]byte(creds.ClientSecret)))
 	os.Setenv("AZURE_SUBSCRIPTION_ID_B64", base64.StdEncoding.EncodeToString([]byte(creds.SubscriptionID)))
 	os.Setenv("AZURE_TENANT_ID_B64", base64.StdEncoding.EncodeToString([]byte(creds.TenantID)))
-	// TODO Figure out a better way of doing this
```

Comment:
@jadarsie same here

Reply:
I do not remember what I meant. Probably something related to the fact that `stdout` is turned into `string` and `[]byte` back and forth a few times.

Reply:
I think it can be removed